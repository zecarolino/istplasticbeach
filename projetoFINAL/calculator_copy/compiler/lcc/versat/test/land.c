int x = 1, y = 2;
int main() { return x && y; }
