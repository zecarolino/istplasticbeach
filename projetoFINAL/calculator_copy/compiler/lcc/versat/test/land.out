VCD info: dumpfile xtop.vcd opened for output.
1
Catched Trap at time  443 clock cycles:
